### PR TITLE
Deprecate scalability tests (we use ClusterLoader already)

### PR DESCRIPTION
**Pull request by wojtek-t**
@kubernetes/sig-scalability-misc @shyamjvs

### CODE REVIEW

File: test/e2e/scalability/density.go [hunk 1]
```diff
@@ -14,6 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// DEPRECATED.
+// We already migrated all periodic and presubmit tests to ClusterLoader.
```
Comment (mm4tt) [inline test/e2e/scalability/density.go]: More of a question than change request.
There seems to be confusion between ClusterLoader and ClusterLoader2, I think we commonly refer to ClusterLoader2 as ClusterLoader, even though there exists thing like ClusterLoader and it's something else than ClusterLoader2. 
Should we get rid (or deprecate) of ClusterLoader to make things clear or is it still used somewhere / by someone?

Comment (wojtek-t) [reply test/e2e/scalability/density.go]: Changed to make explicit that we refer to ClusterLoader2.

I agree we should deprecate ClusterLoader - this was originally open-sources by Redhat, so we should probably talk with them though.

**Review by mm4tt — COMMENTED** [review]
/lgtm

**Pull request merged**